### ISSUE TITLE
Text transformations 

### ISSUE THREAD

**Issue by danyaljj** [allenai/natural-instructions#135]
Pretty much any of the text transformations [here](https://github.com/GEM-benchmark/NL-Augmenter/tree/main/transformations) can be described in plain English and hence can be included as part of natural instructions.

**Comment by swarooprm** [comment]
Also, add unsupervised data creation techniques to synthetically create data.

**Comment by swarooprm** [comment]
I am taking this up along with some collaborators.
